Packs/CoreAlertFields/IncidentFields/incidentfield-Process_Execution_Signature.json

```diff
@@ -28,5 +28,17 @@
     "useAsKpi": false,
     "version": -1,
     "fromVersion": "6.5.0",
-    "x2_fields": "action_process_signature_status"
+    "x2_fields": "action_process_signature_status",
+    "Aliases": [
+        
+        {
+            "cliName": "fireeyesignature",
```

Comment:
This field indicators if a files is signed or not
Values are ['Signed'] Not Signed etc.

Reply:
fixed